packages/api/src/cli/codegen/languages/typescript.ts

```diff
@@ -230,7 +231,7 @@ export default class TSGenerator extends CodeGeneratorLanguage {
       sdkSource
         .getImportDeclarations()
         .find(id => id.getText().includes('HTTPMethodRange'))
-        .replaceWithText("import type { ConfigOptions, FetchResponse } from '@api/core'");
+        ?.replaceWithText("import type { ConfigOptions, FetchResponse } from '@api/core'");
```

Comment:
You originally had these as the following but I think optionally chaining these is a little nicer. 

```
const declaration = sdkSource.getImportDeclarations(...).find(...);
if (declaration) declaration.replaceWithText(...);
```

Cool with the original solution if you want me to revert it though.

Reply:
Yeah I suppose this just returns `undefined` and doesn't actually touch the original `sdkSource` variable if it can't find anything right? Works for me!